tgui/packages/tgui/interfaces/BlueSpaceArtilleryControl.js

```diff
@@ -0,0 +1,55 @@
+import { useBackend } from "../backend";
+import { Button, LabeledList, Section, Box, ProgressBar } from "../components";
+import { Window } from "../layouts";
+
+export const BlueSpaceArtilleryControl = (props, context) => {
+  const { act, data } = useBackend(context);
+  return (
```

Comment:
The old nano syntax
```tmpl
	<div class="itemContent">	
		{{if data.ready}}	
			<span class='good'>Ready</span>	
		{{else data.reloadtime_mins || data.reloadtime_secs}}	
			<span class='bad'>{{:data.reloadtime_mins}}:{{:data.reloadtime_secs}}</span>	
		{{else}}	
			<span class='bad'>No cannon connected!</span>	
		{{/if}}		
	</div>
```

Can be replaced with standard JS if you put it before the `return`
```jsx
let alertStatus;
if (data.ready) {
    alertStatus = (
            <Box bold lineHeight={2} color="green">
              Ready
            </Box>
    )
} else if (data.reloadtime_mins || data.reloadtime_secs) {
    alertStatus = (
            <Box bold lineHeight={2} color="red">
              No cannon connected!
            </Box>
    )
} else {
    alertStatus = (
            <Box bold lineHeight={2} color="red">
              {data.reloadtime_mins}:{data.reloadtime_secs}
            </Box>
    )
}
```

And then in your template, you can just do
```jsx
              <LabeledList.Item label="Alert">
                {alertStatus}
              </LabeledList.Item>
```